pkgs/development/python-modules/automx2/default.nix

```diff
@@ -2,11 +2,13 @@
   lib,
   buildPythonPackage,
   fetchFromGitHub,
+  fetchpatch,
   flask,
   flask-migrate,
   ldap3,
   pytestCheckHook,
   pythonOlder,
+  pythonPackages,
```

Comment:
```suggestion
  sdnotify,
```

Reply:
Obsolete. Upstream built their own sdnotify into automx2 since the implementation is quite trivial and they didn't want to introduce another dependency.